lib/StyleDictionary.js

```diff
@@ -352,6 +361,7 @@ export default class StyleDictionary extends Register {
       preprocessedTokens = expandTokens(preprocessedTokens, this.options);
     }
     this.tokens = preprocessedTokens;
+    this.allTokens = flattenTokens(/** @type {PreprocessedTokens} */ (this.tokens), this.usesDtcg);
```

Comment:
moved, used to be the last ran platform specific transformed tokens (not intuitive at all), which doesn't make sense as a global SD prop. Now this prop is the parsed/preprocessed flattened tokens prior to running platforms.